src/overlays/actors/ovl_En_Zog/z_en_zog.h

```diff
@@ -7,10 +7,42 @@ struct EnZog;
 
 typedef void (*EnZogActionFunc)(struct EnZog*, GlobalContext*);
 
+#define ENZOG_GET_F(thisx) ((thisx)->params & 0xF)
+#define ENZOG_GET_FC00(thisx) (((thisx)->params & 0xFC00) >> 0xA)
+
+enum {
+    /* 2 */ ENZOG_F_2 = 2,
+};
+
+enum {
+    /* 63 */ ENZOG_FC00_63 = 63,
+};
```

Comment:
+1 on these being defines. There's no general rule of thumb we can provide, it all depends on the context of the actor. And in this case, enums doesn't really make sense